SU-SWS/saa_victoria_subtheme
# Adapt 682  mobile menu react widget page

# READY FOR REVIEW

# Summary
- Scope scss to only apply on the react widget by nesting scss
- fixing mobile menu style weirdness that only happens on this page https://associates.alumni.stanford.edu/associates-designees
Note: This is the react widget we were given from Alumni and have done minimal work with. I know scss is more efficient and we have one scss file in it, but we aren't redoing their widget at this time so it uses mostly css files.

# Review By (Date)
- 9/22

# Urgency
- high - bad menu appearance on mobile on just this page:  https://associates.alumni.stanford.edu/associates-designees 
but it is ON PRODUCTION

# Steps to Test

1. Pull down the branch or check out https://saaassociates-test.sites.stanford.edu/associates-designees
2. Clear the site cache
3. View the ../node/61 page in mobile (make sure you have this node created to see the React widget.
4. Make sure the mobile menu appears normally 
5. Check that the react widget still functions with filtering and searching.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-682
